Makefile

```diff
@@ -190,7 +192,20 @@ docker-push-manifest: ## Push the fat manifest docker image. TODO: Update bazel
 
 .PHONY: clean
 clean: ## Remove all generated files
-	rm -f bazel-*
+	$(MAKE) clean-bazel
+	$(MAKE) clean-bin
+
+.PHONY: clean-bazel
+clean-bazel: ## Remove all generated bazel symlinks
+	rm -rf bazel-*
```

Comment:
Should this call `bazel clean` or `bazel clean --expunge` instead?

Reply:
Good point, opted for `bazel clean`, given that expunge is like nuking your entire drive 😂